gallery/src/components/NxTextInput/NxTextInputPage.tsx

```diff
@@ -119,9 +119,91 @@ const NxTextInputPage = () =>
             </NxTable.Row>
             <NxTable.Row>
               <NxTable.Cell>
-                HTML <NxCode>&lt;input&gt;</NxCode> Attributes |
-                HTML <NxCode>&lt;textarea&gt;</NxCode> Attributes
+                id
               </NxTable.Cell>
+              <NxTable.Cell>string</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell>
+                An HTML id, which will be applied to the underlying <NxCode>&lt;input&gt;</NxCode> and not the top-level
+                div rendered by this component.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>
+                disabled
+              </NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell>
+                Whether the input is disabled
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>
+                placeholder
+              </NxTable.Cell>
+              <NxTable.Cell>string</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell>
+                The placeholder for the input
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>
+                aria-required
+              </NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell>
+                Whether the <NxCode>&lt;input&gt;</NxCode> will be marked as aria-required.
+                div rendered by this component.
```

Comment:
I don't think we need 'div rendered by this component'
```suggestion
```

Reply:
> I don't think we need 'div rendered by this component'

Probably a copy/paste error